src/liveMarkbookCalculator.js

```diff
@@ -29,9 +29,9 @@ const calculateLayer = layer => {
         /* Update the mark display */
         let row = layer[i].row;
         if (row) {
-            row.find('td:nth-child(2) > input').val(+mark.toFixed(2));
-            row.find('td:nth-child(4) > input').val(weight);
-            row.find('td:nth-child(5) > input').val(markDenom);
+            row.find('td:nth-last-child(4) > input').val(+mark.toFixed(2));
+            row.find('td:nth-last-child(2) > input').val(weight);
+            row.find('td:nth-last-child(1) > input').val(markDenom);
```

Comment:
```suggestion
            row.find('td:nth-child(2) > input').val(+mark.toFixed(2));
            row.find('td:nth-child(4) > input').val(weight);
            row.find('td:nth-child(5) > input').val(markDenom);
```
Revert back to searching from left to right since the percentage column is at the end now.